app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
:red_circle: I think you should disable the confirmable feature as you are not sending an email to confirm. If you keep this feature none will be able to test this app as it will be impossible to log in unless someone update manually the confirm_at column value. That is what I did to be able to test your app.

![login](https://github.com/huzaifakhan0308/Recipe-app/assets/26736582/5e194d05-ba73-47eb-8d0d-ac5633be1c0a)